src/client/components/section-header/section-header.stories.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { withKnobs } from '@storybook/addon-knobs';
+import SectionHeader from './section-header.component';
+
+export default {
+  component: SectionHeader,
+  title: 'Headers',
+  decorators: [withKnobs],
+};
+
+const sectionTitles = ['Crop', 'Stage Details', 'Trays and Tanks', 'Costs'];
+
+export const SectionHeaders = () => {
+  return (
+    <>
+      <div
+        style={{
+          backgroundColor: '#ebecf0',
+          display: 'flex',
+          flexDirection: 'column',
+          height: '20em',
+          justifyContent: 'space-around',
+        }}
+      >
+        {sectionTitles.map((title, index) => {
+          return (
+            <SectionHeader onClick={action('header clicked')} tabIndex={index}>
```

Comment:
Hmm, I pass this `onClick` both to the `div`'s `onKeyDown` and `onClick`  through props. It kind of needs to be in there (see previous comments). :D 

But, as I said, if this will work better as a button, I have no problem with rewriting it. 👍

Reply:
Alright if that is the lint rules we go with then let's allow it 👍